### DISCUSSION TITLE
antdv.com is down

### DISCUSSION THREAD

**Discussion by johnsanders2** [vueComponent/ant-design-vue#5006] [General]
Just wondering if anyone is aware that the ant design vue website is down?

**Comment by johnsanders2** [comment]
Tried on multiple different machines. Website is down on all of them.

**Reply by tangjinzhou** [reply]
It may be a regional issue, you can clone and deploy the document

**Reply by johnsanders2** [reply]
Can you provide any instruction? Where is the repository to clone from? I don't see any doc references in this Github.

**Comment by sneakylenny** [comment]
It's for my region as well
Country: Netherlands
I use a VPN to access the site

**Reply by johnsanders2** [reply]
Still down for me today as well after 5 days.
I'm in the US.

**Reply by sneakylenny** [reply]
I figured out why it was down for me. The organization where I work has A very restricted Meraki firewall that blocked me from accessing the site.

**Comment by johnsanders2** [comment]
![image](https://user-images.githubusercontent.com/83040804/146049906-0a2aee23-0f9b-4dc8-9363-675575cdcaa6.png)

**Reply by johnsanders2** [reply]
This is the same message I've been getting for 5 days now.
